atomicapp/providers/openshift.py

```diff
@@ -133,6 +146,49 @@ def process_template(self, url, template):
             logger.error(msg)
             raise ProviderFailedException(msg)
 
+    def execute(self, namespace, pod, container, command):
+        """
+        Execute a command in a container in an Openshift pod.
+
+        Args:
+            namespace (str): Namespace
+            pod (str): Pod name
+            container (str): Container name inside pod
+            command (str): Command to execute
+
+        Returns:
+            Command output (str)
+        """
+        args = {
+            'token': self.access_token,
+            'namespace': namespace,
+            'pod': pod,
+            'container': container,
+            'command': ''.join(['command={}&'.format(word) for word in command.split()])
+        }
+        url = 'wss://' + urljoin(
+            self.kubernetes_api,
+            'namespaces/{namespace}/pods/{pod}/exec?'
+            'access_token={token}&container={container}&'
+            '{command}stdout=1&stdin=0&tty=0'.format(
+              **args)).split('://', 1)[-1]
```

Comment:
So part of what we are doing here is removing `ttps ://` from `self.kubernetes_api`
Can you comment on what you are doing. Even better may be to comment and break it up into two lines.